applications/serial_lte_modem/doc/DFU_AT_commands.rst

```diff
@@ -159,12 +159,16 @@ Response syntax
 
 ::
 
-  XDFUSIZE: <file_size>,<download_size>
+  XDFUSIZE: <file_size>,<download_size>,<crc32_checksum>
 
 * The ``<file_size>`` is an integer.
   It indicates the size of the DFU image file.
 * The ``<download_size>`` is an integer.
   It indicates the size that has been downloaded so far.
+* The ``<crc32_checksum>`` is a 32-bit unsigned integer.
+  It indicates the IEEE CRC32 of the image content that has been downloaded.
+  You can use this CRC32 to check data integrity when you use MCUboot to perform a DFU.
+  The CRC checksum should be checked right after #XDFURUN has transferred the downloaded data image from nRF91 to nRF5x, before configuring MCUBoot to use the new image.
```

Comment:
```suggestion

  You should check the CRC checksum after #XDFURUN has transferred the downloaded data image from nRF91 to nRF5x, before configuring MCUboot to use the new image.
```
Better to have the reader as the active person instead of the checksum.
Note also the empty line before, to separate the text a bit from the preceding lines.